### DISCUSSION TITLE
✅ Dark Mode

### DISCUSSION THREAD

**Discussion by itr6** [isontheline/pro.webssh.net#171] [Ideas]
Are there any plans to add in a dark mode for the entire app so the main menu when selecting a server isn’t blinding white?

Thanks, and love the app!

**Comment by isontheline** [comment]
Thank you a lot for your support @itr6 👍

Yes dark mode is planned for the next release (if no huge bugs are coming) : https://github.com/isontheline/pro.webssh.net/issues/7

Have a nice day ☀️

**Reply by isontheline** [reply]
I'm working actively on it

**Comment by isontheline** [comment]
Dark mode is supported since WebSSH 14.11 - Milky Way 🌌 - March 1, 2021
